getting-started/python/index.md

```diff
@@ -46,42 +44,42 @@ is your central gateway to
 
 ### Collections
 
-The EO data available at a back-end is organised in so-called collections.
-For example, a back-end might provide fundamental satellite collections like "Sentinel 1" or "Sentinel 2",
+The Earth observation data (the input of your openEO jobs) is organised in so-called collections,
+e.g. fundamental satellite collections like "Sentinel 1" or "Sentinel 2",
 or preprocessed collections like "NDVI".
-Collections are used as input data for your openEO jobs.
 
 ::: tip Note
 More information on how openEO "collections" relate to terminology used in other systems can be found in
 [the openEO glossary](https://openeo.org/documentation/1.0/glossary.html#eo-data-collections).
 :::
 
 
-Let's list all available collections on the back-end,
-using [`list_collections`](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.connection.Connection.list_collections):
+While it's recommended to browse the available EO collections on the  
+[openEO Platform collections overview webpage](../../data-collections/index.md),
+it's possible to list and inspect them programmatically, 
+as very simple example of openEO Python client usage,
+using methods on the `connection` object we just created 
+(like [`list_collection_ids`](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.connection.Connection.list_collection_ids))
+or [`describe_collection`](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.connection.Connection.describe_collection)):
 
-```python
-print(connection.list_collections())
 ```
+>>> # Get all collection ids
+>>> print(connection.list_collection_ids())
+['AGERA5', 'SENTINEL1_GRD', 'SENTINEL2_L2A', ...
 
-which returns list of collection metadata dictionaries, e.g. something like:
+>>> # Get a listing of all collections with metadata
+>>> print(connection.list_collections())
```

Comment:
I think we should completely drop the `print(connection.list_collections())` example,
it's a huge JSON dump that spams your interactive notebook/session and is practically unusable for a novice user

Reply:
Agreed.

Reply:
It is still included, isn't it?